storybookjs/storybook
# Theming: legacy options don't override set config options

Issue: closes #6521

## What I did

Ensure to only `applyDeprecatedLayoutOptions` if the deprecated options exist and if they do exist do not override them

## How to test

- Is this testable with Jest or Chromatic screenshots? yeah 
- Does this need a new example in the kitchen sink apps? no
- Does this need an update to the documentation? no

If your answer is yes to any of these, please make sure to include it in your PR.
